lib/srv/forward/sshserver.go

```diff
@@ -486,6 +486,12 @@ func (s *Server) GetHostUsers() srv.HostUsers {
 	return nil
 }
 
+// GetHostSudoers returns the HostSudoers instance being used to manage
+// sudoer file provisioning, unimplemented for the forwarder server.
+func (s *Server) GetHostSudoers() srv.HostSudoers {
+	return nil
```

Comment:
We never check explicitly check the return value of GetHostSudoers() for a `nil` return value, from what I can tell; should the forwarding server return a no-op implementation of `HostSudoers` that always returns errors instead?

Reply:
I think i'd prefer GetHostSudoers to return nil, notimplemented from the forwarder